site/public/css/server.css

```diff
@@ -393,8 +393,15 @@ table tr.table-header {
     font-size: 20px;
 }
 
-#header-text>h2:nth-child(2) {
-    padding-top: 25px;
+#header-text>div {
+    display: block;
+    font-size: 1.5em;
+    font-weight: bold;
+
+    /* vertical centering */
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
```

Comment:
Playing around with this, it looks like the positioning stuff is unnecessary and that removing it would leave the content in the exact same place?

Reply:
Actually, it looks worse with these lines of code.  If the gradeable title is really long, and overflows a narrow page, with these lines, the full breadcrumb is not visible.  Without these lines it is all visible.  

with:
<img width="538" alt="Screen Shot 2019-04-22 at 10 46 52 PM" src="https://user-images.githubusercontent.com/5871417/56549882-09f0a400-6552-11e9-88af-dc919e0f3de8.png">

without:
<img width="543" alt="Screen Shot 2019-04-22 at 10 57 20 PM" src="https://user-images.githubusercontent.com/5871417/56549901-11b04880-6552-11e9-9099-a7393bb9d011.png">

Reply:
So I've removed it.  It can be added back in (w/ modifications) if necessary.